connectors/services/job_scheduling.py

```diff
@@ -91,10 +91,13 @@ async def _schedule(self, connector):
         data_source = source_klass(connector.configuration)
         data_source.set_logger(connector.logger)
 
-        connector.log_debug("Validating configuration")
         try:
+            connector.log_debug("Validating configuration")
             data_source.validate_config_fields()
             await data_source.validate_config()
+
+            connector.log_debug("Pinging the backend")
+            await data_source.ping()
```

Comment:
Should we try/catch for a NotImplementedError? And just skip past it if it's not implemented? https://github.com/elastic/connectors/blob/main/connectors/source.py#L557-L562

Reply:
Do we actually have data sources, which do not implement `ping`? Otherwise I would either keep it like that or log a clear error that `ping` is missing, if we expect it from every data source.

Reply:
We currently have no sources that do not implement ping.

I copied the logic from https://github.com/elastic/connectors/blob/main/connectors/sync_job_runner.py#L129-L134